app/models/user.rb

```diff
@@ -5,7 +5,13 @@ class User < ActiveRecord::Base
   devise :database_authenticatable, :registerable,
          :recoverable, :rememberable, :trackable, :validatable,
          :omniauthable, :omniauth_providers => [:facebook]
- 
+
+  has_many :user_movies
+  
+  def watchlist
```

Comment:
If they don't have anything on their watchlist, it will return an empty ActiveRecord collection.